src/mkdocstrings_handlers/python/handler.py

```diff
@@ -99,6 +99,7 @@ class PythonHandler(BaseHandler):
         "members": None,
         "filters": ["!^_[^_]"],
         "annotations_path": "brief",
+        "load_external_modules": False,
```

Comment:
You're documenting the `preload_external_modules` but are using `load_external_modules` in the config?